app/default_overrides_test.go

```diff
@@ -80,11 +80,11 @@ func TestDefaultConsensusConfig(t *testing.T) {
 			Size:                  tmcfg.DefaultMempoolConfig().Size,
 			WalPath:               tmcfg.DefaultMempoolConfig().WalPath,
 
-			// overrides
-			MaxTxBytes:   7_897_088,
-			MaxTxsBytes:  39_485_440,
+			// Overrides
+			TTLNumBlocks: 12,
 			TTLDuration:  75 * time.Second,
-			TTLNumBlocks: 5,
+			MaxTxBytes:   8 * mebibyte,
+			MaxTxsBytes:  40 * mebibyte,
```

Comment:
Per sync discussions elsewhere I think we can increase this to 1 GB

Reply:
Created https://github.com/celestiaorg/celestia-app/issues/3966